content/en/docs/languages/java/jmx.md

```diff
@@ -0,0 +1,341 @@
+---
+title: JMX Metrics
+weight: 14
+description: Collect metrics from JMX MBeans using OpenTelemetry
+cSpell:ignore: mbean mbeans jmxremote jconsole
+---
+
+This page describes how to collect metrics from
+[JMX](https://docs.oracle.com/javase/8/docs/technotes/guides/management/agent.html)
+(Java Management Extensions) MBeans and export them to OpenTelemetry.
+
+## Overview
+
+JMX (Java Management Extensions) is a Java technology that provides tools for
+managing and monitoring applications, system objects, devices, and
+service-oriented networks. JMX MBeans (Managed Beans) expose management
+attributes and operations that can be monitored.
+
+The OpenTelemetry JMX Metric Insight module allows you to bridge JMX metrics
+into OpenTelemetry, enabling you to:
+
+- Monitor JVM metrics (memory, garbage collection, threads, etc.)
+- Collect metrics from application-specific MBeans
+- Export JMX data alongside other OpenTelemetry telemetry signals
+- Use predefined metric mappings for popular frameworks
```

Comment:
The "framework" tends to imply it's mostly a library, at least for now this mostly covers application servers.

```suggestion
- Use predefined metric mappings for popular target systems (Tomcat, Jetty, Wildfly, ...)
```